.docker/HomeAssistantStable/compose.yml

```
@@ -0,0 +1,13 @@
+  homeassistant:
+    container_name: homeassistantstable
+    image: "ghcr.io/home-assistant/home-assistant:stable"
+    volumes:
+      - .\config:/config
+      - /etc/localtime:/etc/localtime:ro
+      - /run/dbus:/run/dbus:ro
+    restart: unless-stopped
+    privileged: true
+    #network_mode: host
+    ports:
+      - "8123:8123"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)